schemas/app-sre/automated-action-argument-1.yml

```diff
@@ -0,0 +1,45 @@
+---
+"$schema": /metaschema-1.json
+version: "1.0"
+type: object
+
+additionalProperties: false
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /app-sre/automated-action-argument-1.yml
+  labels:
+    "$ref": "/common-1.json#/definitions/labels"
+  type:
+    type: string
+
+  # type=openshift related properties
+  namespace:
+    "$ref": "/common-1.json#/definitions/crossref"
+    "$schemaRef": "/openshift/namespace-1.yml"
+  kind_pattern:
+    type: string
+    description: "The kind pattern to match. Default: Deployment|Pod"
```

Comment:
I really prefer to be very "pythonic" and avoid defaults here. It will make app-interface configuration more understandable for those unfamiliar.

Reply:
Do you mean no defaults, and those are required attributes?

Reply:
Done in b5a0e84

Reply:
hmmm, that's for maxOps for `kind_pattern` 😅

Reply:
🤦 done in [0dd81ee](https://github.com/app-sre/qontract-schemas/pull/780/commits/0dd81ee2a1876b7fed553ec9664598963c57f62e)

Reply:
thx!